image/nodejs.sh

```diff
@@ -4,8 +4,13 @@ source /pd_build/buildconfig
 set -x
 
 echo "+ Enabling Node Source APT repo"
-curl -sL https://deb.nodesource.com/setup_18.x | bash -
-apt-get update
+NODE_MAJOR=18
+sudo apt-get update
+sudo apt-get install -y ca-certificates curl gnupg
+mkdir -p /etc/apt/keyrings
+curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | sudo gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg
+echo "deb [signed-by=/etc/apt/keyrings/nodesource.gpg] https://deb.nodesource.com/node_${NODE_MAJOR}.x nodistro main" | sudo tee /etc/apt/sources.list.d/nodesource.list
+sudo apt-get update
```

Comment:
`sudo` is not available/needed and can be removed in all of the above.

Reply:
Removed in b37d17f